provider/resources.go

```diff
@@ -166,6 +166,9 @@ func Provider() tfbridge.ProviderInfo {
 											"account": {
 												MaxItemsOne: tfbridge.True(),
 											},
+											"authentication_key": {
```

Comment:
did you add this manually or was it the upgrade-provider? Here we only define the fields that must be that need to be modified/overrided, like using `AltTypes` to use Enums, or `MaxItemsOne` when we use a list/set in terraform as an object and it does not have MaxItems:1 specified

Reply:
Okay got it. I have added it manually but removed it from the PR.